api-participant/src/main/java/bio/terra/pearl/api/participant/controller/survey/AnswerMappingController.java

```diff
@@ -0,0 +1,64 @@
+package bio.terra.pearl.api.participant.controller.survey;
+
+import bio.terra.pearl.api.participant.api.AnswerMappingApi;
+import bio.terra.pearl.api.participant.service.AuthUtilService;
+import bio.terra.pearl.api.participant.service.RequestUtilService;
+import bio.terra.pearl.core.model.EnvironmentName;
+import bio.terra.pearl.core.model.participant.ParticipantUser;
+import bio.terra.pearl.core.model.survey.AnswerMapping;
+import bio.terra.pearl.core.model.survey.AnswerMappingTargetType;
+import bio.terra.pearl.core.service.exception.NotFoundException;
+import bio.terra.pearl.core.service.survey.AnswerMappingService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class AnswerMappingController implements AnswerMappingApi {
+  private final AnswerMappingService answerMappingService;
+  private final HttpServletRequest request;
+  private final RequestUtilService requestUtilService;
+  private final AuthUtilService authUtilService;
+  private final ObjectMapper objectMapper;
+
+  public AnswerMappingController(
+      AnswerMappingService answerMappingService,
+      HttpServletRequest request,
+      RequestUtilService requestUtilService,
+      AuthUtilService authUtilService,
+      ObjectMapper objectMapper) {
+    this.answerMappingService = answerMappingService;
+    this.request = request;
+    this.requestUtilService = requestUtilService;
+    this.authUtilService = authUtilService;
+    this.objectMapper = objectMapper;
+  }
+
+  @Override
+  public ResponseEntity<Object> findAnswerMapping(
```

Comment:
I don't think we need this controller at all.  The pre-enrollment survey should contain the question show/hide logic within itself.   Assuming you add the "proxyProfile" surveyJs variable we talked about this afternoon, the preEnrollment survey can just have prepopulate/readOnly logic based on whether that proxyProfile variable exists.  See the `enableIf` property on surveyJs questions

Reply:
Hmm, I see what you mean. My goal was to minimize how many "magic" json strings you had to include for proxy enrollment that required us to add documentation and might be points of confusion for study managers trying to add their own proxy enrollment questions, especially since this was a behavior I can't come up with a reason why someone would want to disable and might lead to some confusion when people forget to include them. But I can see how the surveyjs approach requires making fewer internal concepts visible, so I can go ahead and remove it.